src/main/front/src/components/FeedRecommend.jsx

```
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import { Typography, Paper, Box } from "@mui/material";
+import FeedRecommendSlider from "./FeedRecommendSlider";
+import TagChipList from "./TagChipList";
+import useLoadData from "../hooks/useLoadData";
+import GoToAllFeedButton from "./GoToAllFeedButton";
+
+function FeedRecommend({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)